sysutils/simplotask/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/umputun/spot 1.8.1 v
+name                simplotask
+revision            0
+categories          sysutils
+maintainers         {@sikmir disroot.org:sikmir} openmaintainer
+license             MIT
+
+description         A tool for effortless deployment and configuration management
+long_description    {*}${description}
+homepage            https://simplotask.com
+
+build.cmd           make
+build.pre_args-append \
+                    REV=v${version}
+build.args          build
+
+checksums           ${distname}${extract.suffix} \
```

Comment:
If you don't have a `go.vendors` section (which you don't), then you have just a single checksum, and don't really need to specify `${distname}${extract.suffix}`

Reply:
Fixed